src/lib/sendMessage.js

```diff
@@ -0,0 +1,28 @@
+import lineClient from 'src/webhook/lineClient';
+import lineNotify from 'src/lib/lineNotify';
+
+const notify = async (token, message) => {
+  lineNotify(token, { message: message });
+};
+
+// https://developers.line.biz/en/reference/messaging-api/#send-multicast-message
+const multicast = async (userIdList, message) => {
```

Comment:
I would suggest we send flex message with button when doing multicast, so that we can hide complex LIFF URLs from users using a button with URI action.